StockBugFix/StockBugsPatches.cs

```diff
@@ -452,5 +453,61 @@ public static class SolidTransferArm_OnSpawn_Patch {
 					storage.SetDefaultStoredItemModifiers(Storage.StandardSealedStorage);
 			}
 		}
+
+		/// <summary>
+		/// Applied to SpaceHeater to fix Tepidizer target temperature area being too large.
+		/// </summary>
+		[HarmonyPatch(typeof(SpaceHeater), "MonitorHeating")]
+		public static class SpaceHeater_MonitorHeating_Patch {
+			/// <summary>
+			/// Transpiles MonitorHeating to replace the GetNonSolidCells call.
+			/// </summary>
+			internal static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> method) {
+				List<CodeInstruction> instructionList = method.ToList();
```

Comment:
Nit: If you use the `List<T>.List(IEnumerable<T>)` constructor, then `System.Linq` can be avoided, like in L248

Reply:
Fixed